### DISCUSSION TITLE
How do i create airfoil and rotor geometry database using propeller geometry file

### DISCUSSION THREAD

**Discussion by Ajaykrishna0703** [byuflowlab/FLOWUnsteady#158] [General]
Hello everyone
I am a new user of FLOWUnsteady. I have a propeller geometry in STEP format. FLOWUnsteady requires a database of airfoil and rotor geometries to automate the generation of rotors. Can you suggest a way for me to obtain the required data for my propeller geometry using the STEP file?.

**Comment by EdoAlvarezR** [comment]
In the past I've converted the STEP file into STL (just find any converter online), then open the STL in ParaView and slice the blade at multiple sections along the span saving the airfoil contours into CSV files. I would then postprocess those CSV files to calculate the chord distribution, twist distribution, and leading edge position (y and z coordinates), while also untwisting and normalizing the airfoil sections.

**Reply by EdoAlvarezR** [reply]
Related to #162
